crates/moonlight-installer/src/main.rs

```diff
@@ -18,11 +18,9 @@ fn main() -> eframe::Result {
         ..Default::default()
     };
 
-    eframe::run_native(
-        "moonlight installer",
+    Ok(eframe::run_native(
+        "moonlight installer 🌙🌌",
```

Comment:
Not a huge fan of the emojis - could we get rid of them?

Reply:
oki